src/core/Toast/ToastMaster.tsx

```diff
@@ -6,70 +6,17 @@ import '@itwin/itwinui-css/css/toast-notification.css';
 import React from 'react';
 import Toast, { ToastProps } from './Toast';
 
-export type ToastSettings = Omit<ToastProps, 'isVisible' | 'id'>;
-
 type ToastMasterProps = {
-  addToastHandler: (addToast: (settings: ToastSettings) => void) => void;
-  closeAllHandler: (closeAll: () => void) => void;
+  toasts: ToastProps[];
 };
 
 export const ToastMaster = (props: ToastMasterProps) => {
```

Comment:
Since this component is very basic now, I would rename it to just `ToastWrapper`, aligning it with the class name.

Reply:
renamed